### ISSUE TITLE
[Feature Request] Add (optional) FTP server support

### ISSUE THREAD

**Issue by mllg** [xbianonpi/xbian#364]
Albeit xbian already supports smb and sftp to access its file system and connected devices  via network, I'm missing the possibility to transfer files from and to my raspberry in a CPU friendly way.

The samba daemon utilizes all available CPU while transferring files. Thus, the transfer speed is limited by available CPU. Furthermore, XBMC is quite unresponsive while transferring files (video playback works, though). 

I've observed the same CPU intensiveness of smbd using Gentoo and Arch on the RPi and switched to ftp. This resulted in doubled transfer rates (~4 MB/s to >8 MB/s) while using clearly less CPU. Moreover, ftp is in comparison to smb/nfs/sftp nicely integrated in all three major operating systems.

Long story short: It would be great to see a "configuration-free" xbian-package-*ftpd package in a future release.
